auditors/netpols/fixtures/namespace-allow-missing-default-deny-ingress-old-label.yml

```diff
@@ -0,0 +1,20 @@
+# this is to test backwards compatibility with old unregistered annotations (kubernetes.io)
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: namespace-missing-default-deny-ingress-netpol-allowed
+  labels:
+    audit.kubernetes.io/namespace.allow-non-default-deny-ingress-network-policy: "SomeReason"
```

Comment:
tests namespace override labels for the old annotation, so we make sure it's all still working